backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -570,6 +566,18 @@ export class UserResolver {
     return true
   }
 
+  @Authorized([RIGHTS.QUERY_OPT_IN])
+  @Query(() => Boolean)
+  async queryOptIn(@Arg('optIn') optIn: string): Promise<boolean> {
+    const optInCode = await LoginEmailOptIn.findOneOrFail({ verificationCode: optIn })
```

Comment:
don't fail but return `false`?

Reply:
No I catch the Fail in the frontend and send the user to the forgot-password page in this case.
But we can discus this!